backend/src/main/java/co/inventorsoft/scripty/ScriptyApplication.java

```diff
@@ -2,6 +2,9 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import springfox.documentation.swagger2.annotations.EnableSwagger2;
+
+@EnableSwagger2
```

Comment:
Since it Swagger's specific annotation and we have separate config file for Swagger config, let's move annotation there